src/index.js

```diff
@@ -0,0 +1,59 @@
+import './style.css';
+import { saveTasks } from './modules/localStorage.js';
+import Tasks from './modules/tasks.js';
+import newTask from './modules/newTask.js';
+import clearCompletedTasks from './modules/clearCompletedTasks.js';
+import deleteTask from './modules/deleteTask.js';
+import editTask from './modules/editTask.js';
+
+const addTaskBtn = document.querySelector('#addTaskBtn');
+const refreshBtn = document.querySelector('#refreshBtn');
+const tasks = new Tasks();
+
+// display all tasks which are stored in the local storage
+tasks.display();
+
+// Add new task
+const newTaskForm = document.querySelector('[data-new-task-form]');
+newTaskForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  newTask(tasks);
+});
+addTaskBtn.addEventListener('click', () => {
```

Comment:
I suggest that you listen to the submit event on the form instead of the click event of the button as if the button is clicked when the required fields in the form are empty, the new task may be added with empty strings to your local storage